spec.bs

```diff
@@ -4227,6 +4233,12 @@ A signal base value is one of the following:
 : "<dfn><code>percent-ig-storage-quota-used</code></dfn>"
 :: Percentage of [=max interest groups total size per owner=] used by the given buyer,
   capped to 110.
+: "<dfn><code>cumulative-buyer-time</code></dfn>"
+:: Time spent by the buyer from their cumulative buyer timeout limit; if the limit has been exceeded
+  returns the limit + 1000. 0 if this is a seller, or no cumulative buyer timeout is configured.
+: "<dfn><code>percent-igs-cumulative-timeout</code></dfn>"
+:: Percentage of participating interest groups that did not get to participate due to a cumulative
+  buyer timeout occurring.

```

Comment:
```suggestion
:: Percentage of participating interest groups that did not get to participate in the auction due to a
  cumulative buyer timeout occurring.
```

Reply:
Done and rephrased further to not have two notions of "participating" in the same sentence.